src/main/java/campus/Campus.java

```diff
@@ -0,0 +1,54 @@
+package campus;
+
+import campus.infrastructure.Parser;
+import campus.infrastructure.Storage;
+import campus.infrastructure.TaskList;
+import campus.infrastructure.Ui;
+
+/**
+ * Contains the logic for the ChatBot named 'Campus'
+ */
+public class Campus {
+    private final Ui ui;
+    private final Parser parser;
+
+    /**
+     * Creation of a Campus instance
+     */
+    @SuppressWarnings("checkstyle:AbbreviationAsWordInName")
+    public Campus() {
+        String FILE_PATH = "src/main/java/campus/dataTest.txt";
+        this.ui = new Ui();
+        TaskList taskList = new TaskList();
+        Storage storage = new Storage(FILE_PATH);
+        this.parser = new Parser(this.ui, taskList, storage);
+    }
+
+    public static void main(String[] args) {
+        new Campus();
+    }
```

Comment:
Is the description on how to launch the program still accurate? Since this main function looks like it wouldn't do anything.